### ISSUE TITLE
Issue#1: Log-In Problem onto DNIF Web Console

### ISSUE THREAD

**Issue by PRASHANT-SAWANT** [dnif/DigiVigi#18]
It happens sometimes that, when logging in to the DNIF Web Console, we are sometimes not able to do it in first time. After successive attempts it happens, but the number attempts is unknown - so we keep trying.

There seems to be a minor glitch of which we aren't aware of. Can someone enlighten?

@Sharbanibasu23 @aakratisahu @shreyaskulkarni412

**Comment by Sharbanibasu23** [comment]
Hi Prashant,

Thanks for picking up this  issue. I faced this issue this week. Let me share the workaround which I have used. This problem happens when system clock is not synced with google. Then two factor authentication does not work. It gives an incorrect code error. Here are the steps for solving this issue -

1. Go to Authenticator plugin settings

2. Click on 'sync clock with Google'

This will solve the issue. Check this link for two-factor authentication problems.
[https://poloniex.freshdesk.com/support/solutions/articles/1000227645-my-two-factor-codes-aren-t-working-incorrect-code-errors-](url)

![authenticator1](https://user-images.githubusercontent.com/38433265/40850848-51f1fb6a-65e3-11e8-80a0-8aa765ccfc14.PNG)
![authen2](https://user-images.githubusercontent.com/38433265/40850867-5e3831fa-65e3-11e8-96fc-e4477428affd.PNG)
